[Cortex XDR] Fix Error Message (#28194)
demisto/content: Packs/Core/ReleaseNotes/2_0_2.md

#### Integrations

##### Investigation & Response

> - Fixed an issue where the integration failed when searching for a user, host, group, or role that did not exist in the following commands:

Reviewer: ```suggestion
- Fixed an issue where the integration failed when searching for a value that does exist, when running one of the following commands:
```

> - Fixed an issue where the integration failed when searching for a value that does exist, when running one of the following commands:
>     - ***core-list-risky-users***
>     - ***core-list-risky-hosts***
>     - ***core-list-user-groups***
>     - ***core-list-roles***

Reviewer 2: ```suggestion
Fixed an issue where the integration failed when searching for a value exists, when running one of the following commands:
- ***core-list-risky-users***
- ***core-list-risky-hosts***
- ***core-list-user-groups***
- ***core-list-roles***
```

Author: this is typo issue, i mean *doesn't*  exist
